docs/src/parameter_distributions.md

```diff
@@ -218,9 +217,82 @@ The pdf of the Normal distribution and its transform to the physical, constraine
 p = plot(p1, p2, legend=false, size = (900, 450)) #hide
 ```
 
-### Multidimensional example
+### [Sample-based distribution](@id samples-example)
+
+We repeat the work of [Simple example revisited](@ref), but now assuming that to create our prior, we only have samples given by the histogram:
+
+```@setup example_three
+# instead of importing ParameterDistributions & dependencies to call constructor,
+# which would make docs build longer and more fragile, simply hard-code Normal()
+# parameters found by constrained_gaussian constructor
+
+using Distributions
+using Plots
+Plots.default(lw=2)
+
+N = 5000
+
+#bounded in [0.0, 1.0]
+transform_unconstrained_to_constrained(x) = 1.0 - 1.0 / (exp(x) + 1)
+samples = rand(Normal(0.957711, 0.78507), N)
+constrained_samples = transform_unconstrained_to_constrained.(samples)
+
+p3 = histogram(constrained_samples, bins=50) 
+vline!([0.7]) 
+title!("Prior of samples")
+```
+```@example example_three
+p = plot(p3, legend=false, size = (450, 450)) #hide
+```
+Imagine we **do not know** this distribution is bounded. To create a `ParameterDistribution` one can take a matrix `constrained_samples` whose columns are this data:
+```julia
+distribution = Samples(constrained_samples)
+constraint = no_constraint()
+name = "point_seven"
+prior = ParameterDistribution(distribution, constraint, name)
+```
+!!! note
+    This naive implementation will not enforce any boundaries during the algorithm implementation.
+
+Imagine that we **know** about the boundedness of this distribution, then, as in [Simple example revisited](@ref), we define the constraint
+```julia
+constraint = bounded(0, 1)
+```
+which creates a transformation, allowing us to map the samples into an unbounded space, giving the following histogram:
+```@setup example_four
+# instead of importing ParameterDistributions & dependencies to call constructor,
+# which would make docs build longer and more fragile, simply hard-code Normal()
+# parameters found by constrained_gaussian constructor
+
+using Distributions
+using Plots
+Plots.default(lw=2)
+
+N = 5000
+
+# bounded in [0.0, 1.0]
+# transform_unconstrained_to_constrained(x) = 1.0 - 1.0 / (exp(x) + 1.0)
+ transform_constrained_to_unconstrained(x) = log(1.0 / (1.0 - x) - 1.0)
+unconstrained_samples = rand(Normal(0.957711, 0.78507), N)
+
+p3 = histogram(unconstrained_samples, bins=50) 
+vline!([transform_constrained_to_unconstrained(0.7)]) 
+title!("Prior of samples")
+```
+```@example example_four
+p = plot(p3, legend=false, size = (450, 450)) #hide
+```
+As before we define a `Samples` distribution from matrix whose columns are the (now unconstrained) samples, along with a name to create the `ParameterDistribution`.
+```julia
+distribution = Samples(unconstrained_samples)
```

Comment:
can you include in this snippet the transformation? Something like `unconstrained_samples = constraint.constrained_to_unconstrained.(constrained_samples)`, or maybe it can be written better.